cmd/cli/uninstall_mesh.go

```diff
@@ -199,6 +182,63 @@ func (d *uninstallMeshCmd) run() error {
 			fmt.Fprintf(d.out, "OSM namespace CANNOT be deleted in a managed environment\n")
 		}
 	}
+
+	err := d.deleteClusterResources()
+	return err
+}
+
+func (d *uninstallMeshCmd) findFlagSpecifiedMesh(meshInfoList []meshInfo) (bool, bool) {
+	meshFlagSpecified := d.meshName != ""
+	specifiedMeshFound := false
+	if !meshFlagSpecified {
+		return meshFlagSpecified, specifiedMeshFound
```

Comment:
you know the values here, prefer a return with the direct values not the vars.

`return false, false`

although, with above comment, we can simply return false